signer.go

```diff
@@ -64,14 +64,15 @@ func signHost(ca tls.Certificate, hosts []string) (cert tls.Certificate, err err
 			template.IPAddresses = append(template.IPAddresses, ip)
 		} else {
 			template.DNSNames = append(template.DNSNames, h)
+			template.Subject.CommonName = h
```

Comment:
Why do you need to set the CommonName?
Isn't that deprecated in new certificates?
It doesn't seem like CreateCertificate uses CommonName.

Reply:
Some softwares (e.g., okhttp) still check CommonName when verifying cert.
This is for forward compatibility.